### PR TITLE
build: support webpack target

### PR DESCRIPTION
**Pull request by iantanwx**
This PR adds support for the `bundler` target of [`wasm-pack`](https://rustwasm.github.io/docs/wasm-pack/commands/build.html#target) that causes `wasm-bindgen` to output a Webpack-friendly entrypoint and addresses #11.

Note that at this time, `wasm-pack` does not support multiple targets -- see https://github.com/rustwasm/wasm-pack/pull/705. This change therefore produces two outputs as suggested by other users in that issue.

### CODE REVIEW

**Comment by kwonoj** [comment]
1. I would use `exports` field if it's possible since it's officially supported way in node.js, similar to https://github.com/sveltejs/svelte/blob/main/packages/svelte/package.json#L20 . afaik latest webpack also honors these export condition.

2. My main dev environment is Windows, so having posix only script unfortunately breaks my daily workflow.

**Comment by iantanwx** [comment]
> I would use exports field if it's possible since it's officially supported way in node.js, similar to https://github.com/sveltejs/svelte/blob/main/packages/svelte/package.json#L20 . afaik latest webpack also honors these export condition.

Sure, will do.

> My main dev environment is Windows, so having posix only script unfortunately breaks my daily workflow.

Would a node.js script work?

**Comment by iantanwx** [comment]
@kwonoj I've converted the script to node.js and added `exports`. I left `module` and `main` for legacy runtimes.

File: package.json [hunk 1]
```diff
@@ -2,7 +2,8 @@
   "name": "woodpile",
   "version": "0.0.5",
   "description": "SWC AST walker for Javascript",
-  "main": "packages/woodpile/pkg/woodpile.js",
+  "main": "packages/woodpile/pkg/node/woodpile.js",
+  "module": "packages/woodpile/pkg/bundler/woodpile.js",
```
[the remark this answers is not in the document]
Comment (iantanwx) [reply package.json:6]: I don't actually know of any way to make only bundlers point to the `bundler` target.

**Pull request closed without merging**